samples/boards/up_squared/gpio_counter/README.rst

```diff
@@ -0,0 +1,95 @@
+.. _up_squared gpio_counter:
+
+UP Squared
+##########
+
+Overview
+********
+This sample provides an example of how to configure GPIO input and output to
+the UP Squared board.
+
+The sample enables a pin as GPIO input (active high) that triggers the increment
+of a counter (range is 0 to size of counter_pins[]). The counter increments for
+each change from 0 to 1 on HAT Pin 16 (BIOS Pin 19). The value of the counter is represented
+on GPIO output (active high) as a 4-bit value
+(bin 0, 1, 2, 3 -> HAT Pin 35, 37, 38, 40).
+
++------------+-----------------------------+
+|  Element   |   Mapping (by column)       |
++============+=============================+
+| Bit (bin)  | n/a |   3 |   2 |   1 |   0 |
++------------+-----+-----+-----+-----+-----+
+| HAT Pin    |  16 |  40 |  39 |  37 |  35 |
++------------+-----+-----+-----+-----+-----+
+| BIOS Pin   |  19 |  38 |  27 |  15 |  14 |
++------------+-----+-----+-----+-----+-----+
+| Direction  |  IN | OUT | OUT | OUT | OUT |
++------------+-----+-----+-----+-----+-----+
+| Active     |   H |   H |   H |   H |   H |
++------------+-----+-----+-----+-----+-----+
+
+
+Requirements
+************
+
+The application requires an UP Squared board connected to the PC through USB
+for serial console. The BIOS settings must be updated as specified in the
+source code comments for HAT Configurations (see table above).
+
+
+References
+**********
+
+- :ref: 'up_squared' board documentation
```

Comment:
change to:  ``:ref:`up_squared` ``  (need to use back-ticks not apostrophes, and no extra spaces)